platform/darwin/test/MGLOfflineStorageTests.mm

```diff
@@ -208,8 +207,8 @@ - (void)testBackupExclusion {
                                                              appropriateForURL:nil
                                                                         create:NO
                                                                          error:nil];
-    // Unit tests don't use the main bundle; use com.mapbox.ios.sdk instead.
-    NSString *bundleIdentifier = [NSBundle bundleForClass:[MGLMapView class]].bundleIdentifier;
+    // As of iOS SDK 12.2 unit tests now have a bundle id: com.apple.dt.xctest.tool
+    NSString *bundleIdentifier = [NSBundle mgl_applicationBundleIdentifier];
```

Comment:
This variable wasn’t necessarily set to the main bundle to begin with – it’s the more robust approach of getting the bundle that contains a specific class.